### PR TITLE
Feedback page accessibility

### PR DESCRIPTION
**Pull request by jcfilben**
Solved all accessibility violations on the feedback page. On the home page github is showing a lot of additions and deletions because I added a `<Main>` section around the content to fix an accessibility violation.

### CODE REVIEW

**Comment by jcfilben** [comment]
I was messing around with the buttons on the feedback page. I set the buttons on the left to be the same size as the button on the right. Here is two options for the margin:

The first one has a 'xsmall' margin like the button on the right but it makes the top 2 buttons take up 2 lines:
Screen Shot 2020-07-23 at 4 19 58 PM

The second option doesn't have the 'xsmall' padding but the 'publisher/github' button is a little tight:
Screen Shot 2020-07-23 at 4 19 30 PM

@ShimiSun Let me know if one of these works or if there is something else I should do instead

**Comment by ShimiSun** [comment]
Let's discuss it on our call tomorrow

File: src/pages/Feedback/Feedback.js [hunk 1]
```diff
@@ -98,14 +101,22 @@ const Feedback = (props) => (
...
-                  />
+                <Card
+                  key={item.label}
+                  onClick={() => {
+                    window.open(item.link, '_self');
+                  }}
+                  background="white"
+                  pad="xsmall"
+                  direction="row"
+                  // width='small'
```
Comment (ShimiSun) [inline src/pages/Feedback/Feedback.js:112]:
```suggestion
```

File: src/pages/Feedback/Feedback.js [hunk 2]
```diff
@@ -121,14 +132,22 @@ const Feedback = (props) => (
...
-            />
+          <Card
+            onClick={() => {
+              window.open('http://slackin.grommet.io/', '_self');
+            }}
+            background="white"
+            pad="xsmall"
+            direction="row"
+            width="small"
+            fill={false}
```
Comment (ShimiSun) [inline src/pages/Feedback/Feedback.js:143]: Is this really needed?

**Comment by ShimiSun** [comment]
This PR needs to resolve conflicts.

**Comment by jcfilben** [comment]
Fixed merge conflicts

**Pull request merged**